packages/cryptoplease_link/lib/src/payments/receive_payment.dart

```diff
@@ -18,7 +19,12 @@ Future<SignedTx> receivePaymentTx({
     commitment: commitment,
   );
 
-  if (escrow == null) throw Exception('Escrow account not found');
+  if (escrow == null) {
```

Comment:
If `escrow` is `null` does it necessarily mean that the payment is already used?